plugins/jobtype/jobtypes/teradataToHdfs/plugin.properties

```diff
@@ -0,0 +1,5 @@
+mr.listener.visualizer=false
+
+#Below is to use latest avro jar for TDCH
```

Comment:
Below two jars will be used when initializing TDwallet on job execution runtime.
plugins/jobtype/jobtypes/hdfsToTeradata/lib/teradata-connector-1.4.1.jar
plugins/jobtype/jobtypes/teradataToHdfs/lib/teradata-connector-1.4.1.jar

I've put the same jar into extlib to make compiler happy on the  build time where in runtime, it will be using the one in lib folder.

To make it ideal, I was thinking about adding plugins' lib folder into classpath, but it seems more work to make sure that other job type won't fail.

Any suggestion is welcome to make it located in one place.